### ISSUE TITLE
Re-enable created and updated fields for master records on the front end

### ISSUE THREAD

**Issue by adrian-lara** [SEED-platform/seed#2119]
### Background
Within SEED, users were presented with created and updated datetime stamps for property and tax lot records.

During the matching revamp work, it was discovered that the datetime stamp was actually coming from the canonical Property and canonical Tax Lot respectively. Ideally, these would be coming from the respective -States models since that is what actually holds the record information that users are seeing. Since linking was not in place, canonical records and connected -State records were 1-to-1, and it wasn't actually an issue.

In preparation for linking, where canonical records might be 1-to-many with -State records, there was an attempt to present users with the created and updated from the -State models instead. To do that, those two fields were both initially created for the -State models, and the canonical versions were disabled. In doing so, the any fields named created and updated were inadvertently prevented from appearing to users.

### Expected Behavior
Created and updated values exist for property and tax lot records and are viewable on the frontend.

### Actual Behavior
Created and updated values for property and tax lot records are no longer viewable.

### Steps to Reproduce
Visit any of the inventory list or inventory detail pages and find that Updated or Created columns are missing. View the column list settings page, and see that these columns are missing as options.

### Instance Information
Server Instance: `prod` and `develop`
Server SHA: `9105437`

**Comment by adrian-lara** [comment]
For `prod`, it was enough to just re-reveal the (canonical) Property and Tax Lot `created` and `updated values. This was possible because, as mentioned above, at the time, that deployment didn't have linking and the updated and created  canonical times was a reasonable stand in for the actual -State values.

**Comment by adrian-lara** [comment]
For `develop`, the fix for this had more complexity and research as we wanted to shift users from the canonical to the -State values. 

Actually, when matching was built out, [the -State values were actually taken from corresponding audit log objects](https://github.com/SEED-platform/seed/commit/efbff203616583d726a526838c5cfabb7df5c754#diff-44e89af066088d7be1e76bd679c1a247). 

Later, the SQL query conducting this copying was inadvertently moved into a different migration: `seed/migrations/0111_rehash.py`. Upon this discovery, it was noticed that this rehashing changed every updated value to whenever that migration was run. 

We [changed the order of when those migrations were run](https://github.com/SEED-platform/seed/pull/2149/files/746c83dcff2c40805abfeb106a4f67107c7a1fed#diff-41b300152661b0f46301c2177e654337).
Also, we decided to add another migration similar to the first (that copied values from audit logs) just for the updated times.
We also took this opportunity to [enhance the migration's query](https://github.com/SEED-platform/seed/pull/2149/files/746c83dcff2c40805abfeb106a4f67107c7a1fed#diff-1f08fab043684cbaf0e89542d986442eR10-R14) to consistently choose the most recent audit log date in the case that a record has 2 audit logs (presumably only happening for records created in older SEED versions).
